StoreApp/Controllers/CartController.cs

```diff
@@ -0,0 +1,178 @@
+                }
+            }
+
+            return PartialView("_CartPartial");
+        }
+
+        [HttpGet]
+        public ActionResult DeleteFromCart(int id)
+        {
+            if (TempData.Peek("Сart") is List<ProductViewModel>)
+            {
+                var cart = TempData.Peek("Сart") as List<ProductViewModel>;
+
+                for (int i = 0; i < cart.Count; i++)
+                {
+                    if (cart[i].Id == id)
+                    {
+                        cart.RemoveAt(i);
+
+                        TempData["Cart"] = cart;
+                    }
+                }
+            }
+
+            return RedirectToAction("Index");
+        }
+
+        [HttpGet]
+        public ActionResult IncreaseProduct(int id)
+        {
+            if (TempData.Peek("Сart") is List<ProductViewModel>)
+            {
+                var cart = TempData.Peek("Сart") as List<ProductViewModel>;
+
+                for (int i = 0; i < cart.Count; i++)
+                {
+                    if (cart[i].Id == id)
+                    {
+                        cart[i].Quantity++;
+
+                        TempData["Cart"] = cart;
+                    }
+                }
+            }
+
+            return RedirectToAction("Index");
+        }
+
+        [HttpGet]
+        public ActionResult DecreaseProduct(int id)
+        {
+            if (TempData.Peek("Сart") is List<ProductViewModel>)
+            {
+                var cart = TempData.Peek("Сart") as List<ProductViewModel>;
+
+                for (int i = 0; i < cart.Count; i++)
+                {
+                    if (cart[i].Id == id)
+                    {
+                        if (cart[i].Quantity > 1)
+                        {
+                            cart[i].Quantity--;
+                        }
+                        else
+                        {
+                            cart.RemoveAt(i);
+                        }
+                        
+                        TempData["Cart"] = cart;
+                    }
+                }
+            }
+
+            return RedirectToAction("Index");
+        }
+
+        [HttpGet]
+        public ActionResult Order()
+        {
+            if (TempData.Peek("Сart") is List<ProductViewModel>)
+            {
+                var cart = TempData.Peek("Сart") as List<ProductViewModel>;
+
+                decimal totalPriceCart = 0;
+
+                foreach (var product in cart)
+                {
+                    totalPriceCart += product.Quantity * product.Price;
+                }
+
+                int orderId = -1;
+
+                if (cart.Count > 0 && totalPriceCart > 0)
+                {
+                    string username;
+                    if (User.Identity.Name == "") username = "user1"; 
+                    else username = User.Identity.Name;
+
+                    int userId = userService.GetAll().First(u => u.UserName.Equals(username)).Id;
```

Comment:
1. Style of **if block** is not correct.
2. As I could understand it's a temporary solution?